modules/ROOT/pages/_partials/intro-con.adoc

```diff
@@ -6,7 +6,7 @@ Red Hat Managed Integration delivers managed integration services hosted on the
 {PRODUCT_SHORT} enables developers adopt event-based computing and API-based development with lightweight, loosely coupled, self-service and flexible components.
 {PRODUCT_SHORT} enables the operations team focus on high value objectives while Red Hat manage the infrastructure. All the applications and services are protected by Red Hat Single Sign-On.
 
-== {webapp}
+== Solution Explorer
```

Comment:
why? does it not resolve?

Reply:
It didn't seem to resolve when I looked at the rendered page. Is that odd?

Reply:
y, let's move on